eng/packages/http-client-csharp/generator/TestProjects/Local/Mgmt-TypeSpec/src/Generated/MgmtTypeSpecClient.RestClient.cs

```diff
@@ -1,14 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-// <auto-generated/>
-
-#nullable disable
-
-namespace MgmtTypeSpec
-{
-    /// <summary></summary>
-    public partial class MgmtTypeSpecClient
```

Comment:
The empty root client needs to be skipped for MPG, it's a bonus to get rid of them in this PR.